src/api/addressApi.js

```diff
@@ -18,26 +18,33 @@ const addressApi = {
     });
   },
 
-  search(address, count, hash, page, token) {
+  getBalance(address, token) {
+    return requestExplorerServiceV1.get(`address/balance`, {params: {address, token}}).then((res) => {
+      if (res && res.data) {
+        return res.data
+      }
+    }).catch((error) => {
+      // something wrong with request
```

Comment:
Missing error treatment

Reply:
Error treatment will be made by the page using the api.